internal/provider/utils.go

```diff
@@ -173,6 +173,17 @@ func listOfInterfacestoStrings(v interface{}) []string {
 	return result
 }
 
+// Converts a list of strings to a list of interfaces
+func listOfStringsToInterfaces(v []string) []interface{} {
+	result := make([]interface{}, len(v))
+
+	for _, s := range v {
+		result = append(result, s)
```

Comment:
I think we'll want something like `result[i] = s` here, right? Otherwise I'm getting `nil`s prepended to the list. But if we keep it a list, we won't need this.

Reply:
@megan07 great catch! was copying and pasting and didn't think about the implications here.